students/zach_meves/lesson08/test_circle.py

```diff
@@ -0,0 +1,359 @@
+"""
+Test circle.py
+"""
+
+import unittest
+import os
+from circle import *
```

Comment:
it's typically bad practice to use star imports, import the whole module: `import circle` or specific classes.